include/game.h

```diff
@@ -192,6 +192,9 @@ typedef struct Primitive {
 #define ANIMSET_DRA(x) (x)
 #define ANIMSET_OVL(x) ((x) | ANIMSET_OVL_FLAG)
 
+#ifndef SOTN_STR
+#define _S(x) (x) // Strings processed by tools/sotn-str.py
```

Comment:
I guess this is long enough since if you grep for `_S(` you will only get these, but if you just search for `_S` there's a lot of hits. Was wondering if it should be a little longer. It's probably fine as-is

Reply:
I'm not totally clear on what this does since i don't know enough about the build chain. It might be nice if there was a writeup about what exactly sotn-str.py does (with an example going from binary to Unicode to the SOTN byte sequence), and why the _S( is necessary since it seems like it's just returning x unchanged. I don't know the details of what is happening here and overall it would probably be nice to have a reference available since I'm sure I'm not the last person who will be confused about what's happening here.

Reply:
Yes, I think so. I think the important thing is that the _S is meaningless in terms of C code, and we're using it as a hack, to be a signal flag that goes to a custom python script that we're inserting in the build chain. Is that a correct understanding?